roadmap.md

```diff
@@ -1,24 +1,29 @@
-# WPF for .NET 5 Roadmap
-WPF is a .NET Core UI framework for building Windows desktop applications. The ownership of WPF was recently transitioned to our team (Developer Ecosystem and Platform team under Windows). This transition enables investments across UI frameworks namely WINUI and WPF to stay aligned and remain future-proof as new technology trends & devices are introduced in the industry.
+# WPF Roadmap FY22
+WPF is a .NET Core UI framework for building Windows desktop applications. WPF is currently supported on the below versions of .NET:
+- .NET Core: .NET 5 or higher (including .NET Core 3.1)
+- .NET Framework: NET Framework 4 and higher
 
-The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2020 and into 2021. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues/new/choose "file a new issue").
+[.NET CORE Roadmap](https://github.com/dotnet/core/blob/main/roadmap.md)
+
+
+The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2021 and into 2022. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues/new/choose "file a new issue").
 
 ### PROPOSED ROADMAP
-| #  | Milestone  |  Target Delivery |
+| #  | Project Title  |  Target Delivery |
 | :------------: | :------------: | :------------: |
-|1  |Incorporating .NET Framework servicing fixes into .NET Core 3.1 and .NET 5 |Ongoing|
-|2  |Ongoing integration with .NET 5 |20H2|
-|3  |Build out Test Infrastructure to add tests to validate and merge community PRs|20H2|
-|4  |Accessibility updates to .NET 5   |21H1|
-|5  |ARM64 Support|21H1|
+|1  |Incorporating .NET Framework servicing fixes into .NET Core 3.1, .NET 5 and .NET 6 |Ongoing|
+|2  |Performance Improvements |Ongoing|
+|3  |Ongoing support and maintenance for .NET Framework, .NET core 3.1, .NET 5.0 and .NET 6.0 |Ongoing|
+|4  |New API review process for Community PRs  |21H2|
+|5  |ARM64 support on .NET 6|21H2|
```

Comment:
The [.NET 6 Preview 1 blog post](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-1/#arm64) states that ARM64 WPF is supported. Is there still more work to be done for ARM64 WPF?

Reply:
Would make sense to just remove this from the roadmap, since it shipped with .NET 6.0